apis/pipeline/pipeline.yaml

```diff
@@ -4693,6 +4693,9 @@ components:
                             items:
                                 type: string
                             example: ["sg-00000xxxx0000xxx1", "sg-00000xxxx0000xxx2"]
+                        useInstanceStore:
+                            description: Setup available instance stores (NVMe disks) to use for Kubelet root if available. As a result emptyDir volumes will be provisioned on local instance storage disks. You can check out available instance storages here https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html#instance-store-volumes.
+                            type: boolean
```

Comment:
Is it intentional the API default value of `useInstanceStore` on a new cluster/node pool is false?
(I vaguely remember discussing it should be opt-out, but I could be mistaking it for something else, so correct me if I'm wrong here, no assumptions just open question.)

(This has an impact on the inlined template parameter initializing logic as well due to default value handling.)

Same at:
https://github.com/banzaicloud/pipeline/pull/3388/files?file-filters%5B%5D=.yaml#diff-d41e335500d6bf91169555b68a970b732b21f70dbecef99585ec49d11f7758b7R4943
(@ 4943 in the API specification, that is the node pool creation/listing argument)

Reply:
Yes it was intentional to set to false by default.